packages/app-admin-ui/client/components/CreateItemModal.js

```diff
@@ -168,6 +168,7 @@ class CreateItemModal extends Component {
                             /* TODO: Permission query results */
                             errors={validationErrors[field.path] || []}
                             warnings={validationWarnings[field.path] || []}
+                            createItemModal={CreateItemModalWithMutation}
```

Comment:
is this casing good, I see that some places Came Case is used when passing component as prop. 
should this be `CreateItemModal={CreateItemModalWithMutation}`?

Reply:
I think the convention is capital when it's a React component so yes probably caps but not a deal-breaker for me.

Reply:
I will fix it anyways to be consistent with other palaces